src/MSBuildBinLogQuery/Component/ItemManager.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
```

Comment:
Could you explain what copyright banners are?

Reply:
Most of our repos require that source code files start with a "banner" that indicates the copyright status of the file, for instance:

https://github.com/dotnet/cli-lab/blob/7caaf2565c41461beba2ff4060dec0c3cd49bed8/src/dotnet-core-uninstall/Program.cs#L1-L2

I don't know what the style in this repo is, but I expected one. @livarcocc?

Reply:
Got it. But currently only `dotnet-core-uninstall/Program.cs` has this banner, since it's auto-generated by Arcade.

Can we merge this PR with this question as a separate issue/PR?